client/my-sites/stats/hooks/use-should-gate-stats.ts

```diff
@@ -128,12 +130,18 @@ export const shouldGateStats = ( state: object, siteId: number | null, statType:
 		}
 
 		const isSiteCommercial = getSiteOption( state, siteId, 'is_commercial' ) || false;
-		// Paywall basic stats for commercial sites.
 		if ( isSiteCommercial ) {
-			return [
-				...jetpackStatsCommercialPaywall,
-				...granularControlForJetpackStatsCommercialPaywall,
-			].includes( statType );
+			const billableMonthlyViews = getPlanUsageBillableMonthlyViews( state, siteId );
+			// Paywall basic stats for commercial sites with monthly views of more than 1k.
+			if ( billableMonthlyViews >= MIN_MONTHLY_VIEWS_TO_APPLY_PAYWALL ) {
+				return [
+					...jetpackStatsCommercialPaywall,
+					...granularControlForJetpackStatsCommercialPaywall,
+				].includes( statType );
+			}
+
+			// Paywall advanced stats for commercial sites with monthly views less than 1k.
+			return [ ...jetpackStatsAdvancedPaywall ].includes( statType );
```

Comment:
👍

Reply:
I imagine the line is added for clarification purpose, which I guess we could omit?

Reply:
I'm happy to keep the line

Reply:
> the line is added for clarification purpose

Exactly! 😄